src/components/Header/UserPreferences/Automation/Details.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import { useAutomation } from '../../../../@context/Automation/AutomationProvider'
+import { useMarketMetadata } from '../../../../@context/MarketMetadata'
+import { useUserPreferences } from '../../../../@context/UserPreferences'
+import Button from '../../../@shared/atoms/Button'
+import Loader from '../../../@shared/atoms/Loader'
+import { AUTOMATION_WALLET_MODES } from '../AutomationWalletMode'
+import Balance from './Balance'
+import styles from './Details.module.css'
+import Import from './Import'
+import Address from './Address'
+import Decrypt from './Decrypt'
+
+function AdvancedView(): ReactElement {
+  const { deleteCurrentAutomationWallet } = useAutomation()
+  const deleteWallet = () => {
+    deleteCurrentAutomationWallet()
+  }
```

Comment:
question: You could directly use deleteCurrentAutomationWallet in the onClick handler without the need for the extra function. Why do we do it this way?

Reply:
Good catch. Agreed. Seems to be leftover from a previous iteration.